public class NoRootWithinFunctionBoundsException extends NumericalMethodException {
  public NoRootWithinFunctionBoundsException() {
   super("NoRootWithinFunctionBoundsException: No root exists within the bounds of the function."); 
  }
}